### PR TITLE
Improve the UX of start_clean.sh #43

### PR DESCRIPTION
**Pull request by liberatr**
* Replace 'Press y to continue' with a more common shell script pattern per #43 
* Fix some of the yellow/green formatting
* Change name of "Site-Install" to "Drupal Sprinting"

### CODE REVIEW

**Comment by rfay** [comment]
Could you please propose this on the other scripts?

**Comment by liberatr** [comment]
@rfay I believe these prompts all use a similar pattern now.

**Comment by joelpittet** [comment]
Big +1, resolves #43

File: install_ddev.sh [hunk 1]
```diff
@@ -74,23 +74,22 @@ if [[ "$OS" == "Darwin" ]]; then
...
-        then
-            printf "${RED}You didn't hit y or Y, exiting script${RESET}"
-            exit 1
-        fi
+        printf "${GREEN}
+####
+# ${YELLOW}Open Docker preferences, confirm its version 18.03.0 and the memory allocation is set to 3.0 GiB${GREEN}
+# ${YELLOW}on the Advanced tab, and that docker has fully restarted before continuing.${GREEN}
+#
+#
```
Comment (BrianGilbert) [inline install_ddev.sh]: Need to remove one more of these #'s please

**Review by BrianGilbert — APPROVED** [review]

**Pull request merged**